### DISCUSSION TITLE
storage s3 driver not deleting files in amazon

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#6757] [Archive]
i try to use s3 driver for storage
uploading is fine
but after delete file stay in amazon, how its fix?

**Comment by Enhed** [comment]
its bug only if added env STORAGE_LOCATION_ROOT
if not add all working fine

**Comment by rijkvanzanten** [comment]
> but after delete file stay in amazon, how its fix?

No idea! If you think there's a bug in the S3 driver, please [open an issue and fill out the form](https://github.com/directus/directus/issues/new?assignees=&labels=Bug+%28Potential%29&template=bug_report.yml).
